apps/prairielearn/src/ee/cron/chunksHostAutoScaling.ts

```diff
@@ -100,9 +100,12 @@ export async function run() {
     (m) => m.Id === LOAD_BALANCER_REQUESTS_PER_MINUTE,
   );
 
-  const maxPageViewsPerSecond = _.max(pageViewsPerSecondMetric?.Values) ?? 0;
-  const maxActiveWorkersPerSecond = _.max(activeWorkersPerSecondMetric?.Values) ?? 0;
-  const maxLoadBalancerRequestsPerMinute = _.max(loadBalancerRequestsPerMinuteMetric?.Values) ?? 0;
+  const maxPageViewsPerSecond = Math.max(...(pageViewsPerSecondMetric?.Values ?? []), 0);
+  const maxActiveWorkersPerSecond = Math.max(...(activeWorkersPerSecondMetric?.Values ?? []), 0);
+  const maxLoadBalancerRequestsPerMinute = Math.max(
+    ...(loadBalancerRequestsPerMinuteMetric?.Values ?? []),
+    0,
+  );
```

Comment:
@nwalters512 I'm assuming in this conversion that all `Values` here are positive, is this a reasonable assumption in this context?

Reply:
I believe it is, yes.